packages/suite/src/hooks/wallet/coinmarket/useCoinmarketInfo.ts

```diff
@@ -27,6 +27,22 @@ function toCryptoOption(cryptoId: CryptoId, coinInfo: CoinInfo): CoinmarketCrypt
     };
 }
 
+const sortPopularCurrencies = (
+    a: CoinmarketCryptoSelectItemProps,
+    b: CoinmarketCryptoSelectItemProps,
+) => {
+    const order = ['bitcoin', 'ethereum', 'litecoin', 'solana', 'cardano'];
```

Comment:
based on?

Reply:
According to earlier sorting based on the backend's response. Now there is no sorting on the backend.

Reply:
I would sort it using order from `networksConfig`. 

cc @Hannsek maybe we should also sort it somehow?
![Screenshot 2024-09-27 at 16 24 12](https://github.com/user-attachments/assets/88322625-a0d7-4313-bad0-488df10c094c)